client/src/components/SignUp.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react';
+import { BrowserRouter, Route, Link, Redirect, Switch } from "react-router-dom";
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+
+import root from '../style/signupLoginPages'
+
+
+const useStyles = makeStyles({
+    root: root
+});
+function SignUp (){
+    const classes = useStyles();
+    return(
```

Comment:
You're missing the state and event handling in this component. Once you add that, it should be good to merge